src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -98,6 +138,28 @@ export const ActionPanelContent = ({
           />
         </ErrorBoundary>
       )}
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        <Button
+          sx={{ width: '95%', mt: '1rem' }}
+          variant="contained"
+          onClick={() => onClickDeleteRegistration(registration)}>
+          {t('common.delete')}
+        </Button>
+      </Box>
```

Comment:
ser ut som knappen er outlined i skissene? blir veldig dominant som contained. Trenger ikke flex når det bare er ett element heller. Egentlig burde vi ikke ha `<Box>` rundt dette heller, men blir noe kluss med bruedden uten det, som jeg ikke ser noen åpenbar løsning på 🤔 

```suggestion
      <Box sx={{ m: '0.5rem', mt: '1rem' }}>
        <Button fullWidth variant="contained" onClick={() => onClickDeleteRegistration(registration)}>
          {t('common.delete')}
        </Button>
      </Box>
```

Reply:
Hadd en prat med Cecilie 16.08, og hun ønsket at den skulle være contained. Men er enig i at den blir veldig fremtredende.